etc/config/cuda.amazon.properties

```diff
@@ -668,11 +668,14 @@ libs.cudacxx.versions.trunk.version=trunk
 libs.cudacxx.versions.trunk.path=/opt/compiler-explorer/libs/libcudacxx/trunk/include
 libs.matx.name=MatX
 libs.matx.description=GPU/CPU-Accelerated Numerical Computing in Modern C++
-libs.matx.options=--expt-relaxed-constexpr
```

Comment:
Looks like we've lost this option - was that intentional?

Reply:
I'll take your word for it: thanks - it may still break people using the links referencing 0.7 library I suppose? But I don't have a good solution. Let's get this merged! thank you